content-learn/mastering-meshery-istio/performance-of-service-mesh/istio/deploy-sample-application.mdx

```diff
@@ -0,0 +1,182 @@
+---
+docType: "Chapter"
+chapterTitle: "Deploy a sample application"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 4
+lectures: 12
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+import BookInfoOffMesh from "../../../../src/assets/images/learn-layer5/istio/bookinfo-off-mesh.png";
+import BookInfoOnMesh from "../../../../src/assets/images/learn-layer5/istio/bookinfo-on-mesh.png";
+
+<ChapterStyle>
+
+To play with Istio and demonstrate some of it's capabilities, you will deploy the example BookInfo application, which is included the Istio package.
+
+<h2 className="chapter-sub-heading">What is the Bookinfo Application </h2>
+<br />
+This application is a polyglot composition of microservices are written in
+different languages and sample BookInfo application displays information about a
+book, similar to a single catalog entry of an online book store. Displayed on
+the page is a description of the book, book details (ISBN, number of pages, and
+so on), and a few book reviews.
+
+The end-to-end architecture of the application is shown in the figure.
+
+<br />
+<a href={BookInfoOffMesh}>
+  <img src={BookInfoOffMesh} width="100%" align="center" />
+</a>
+
+_Figure: BookInfo deployed off the mesh_
+
+<br />
+It’s worth noting that these services have no dependencies on Istio, but make an
+interesting service mesh example, particularly because of the multitude of services,
+languages and versions for the reviews service.
+
+As shown in the figure below, proxies are sidecarred to each of the application containers.
+
+<br />
+<a href={BookInfoOnMesh}>
+  <img src={BookInfoOnMesh} width="100%" align="center" />
+</a>
+
+_Figure: BookInfo deployed on the mesh_
+
+<br />
+Sidecars proxy can be either manually or automatically injected into the pods. Automatic
+sidecar injection requires that your Kubernetes api-server supports `admissionregistration.k8s.io/v1`
+or `admissionregistration.k8s.io/v1beta1` or `admissionregistration.k8s.io/v1beta2`
+APIs. Verify whether your Kubernetes deployment supports these APIs by executing:
+
+```sh
+kubectl api-versions | grep admissionregistration
+```
+
+If your environment **does NOT** supports either of these two APIs, then you may use [manual sidecar injection](https://github.com/layer5io/istio-service-mesh-workshop/blob/master/lab-2/appendix-manual-injection.md) to deploy the sample app.
```

Comment:
There shouldn't be any reference to the `istio-service-mesh-workshop` repo.

Reply:
In that case, we would have to show this as a chapter or what ?

Reply:
Not chapter exactly, this is a sublink ( a header) within a chapter.

Reply:
Ok

Reply:
Will it be ok to keep this under **Manual Installation**, under a heading called **Manual sidecar injection** ?